docs/source/advanced_topics/0_batching.rst

```diff
@@ -1,4 +1,40 @@
 Batching
 ========
 
-TODO
\ No newline at end of file
+Batching is an important function influencing the performance of the pipeline a lot. Inference units utilize batching to process images more efficiently. Primary models (which operate on the whole image) consider the batch as a number of frames simultaneously passed to the model. Secondary models consider the batch as a number of frames simultaneously passed to the model.
```

Comment:
`Secondary models consider the batch as a number of frames simultaneously passed to the model.` -> `Secondary models consider the batch as a number of objects simultaneously passed to the model.`